tools/build_velox.sh

```diff
@@ -71,7 +71,7 @@ if [ $BUILD_VELOX_FROM_SOURCE == "ON" ]; then
     echo "VELOX_INSTALL_DIR=${VELOX_INSTALL_DIR}"
     mkdir -p $VELOX_SOURCE_DIR
     mkdir -p $VELOX_INSTALL_DIR
-    git clone https://github.com/oap-project/velox.git -b main $VELOX_SOURCE_DIR
+    git clone https://github.com/jinchengchenghh/velox.git -b orlist $VELOX_SOURCE_DIR
```

Comment:
remember to modify to 'oap-project'

Reply:
Verified this branch locally on TPC-H and TPC-DS with main branch. Will raise a following PR to change this back since no permission to push to this branch. Thanks.